build/mark-new-version.sh

```diff
@@ -142,13 +148,13 @@ else
   echo "pushed as an ancestor of master, even though the tag is on on a release"
   echo "branch. The tag will thus be found by tools like git describe"
   echo ""
-  echo "- Update so you see that merge in origin"
+  echo "- Update so you see that merge in ${fetch_remote}"
   echo "   git remote update"
   echo "- Create and check out a new branch based on master"
-  echo "   git checkout -b merge-${release_branch}-to-master origin/master"
-  echo "- Merge the ${release_branch} into your merge-${release_branch}-to-master branch"
-  echo "   git merge ${release_branch}"
-  echo "   - It's possible you have merge conflicts."
+  echo "   git checkout -b merge-${release_branch}-to-master ${fetch_remote}/master"
+  echo "- Merge the ${release_branch} into your merge-${release_branch}-to-master branch:"
+  echo "   git merge -s recursive -X ours ${release_branch}"
```

Comment:
That actually works better - I was merely modifying your instructions, which relied on you having a local branch named `release-0.14`, but forcing it from the upstream is probably better, especially since we've done the `git remote update`. Changing.